android/java/org/chromium/chrome/browser/crypto_wallet/util/ParsedTransaction.java

```diff
@@ -252,25 +140,23 @@ public static ParsedTransaction parseTransaction(TransactionInfo txInfo,
         final ParsedTransactionFees feeDetails = ParsedTransactionFees.parseTransactionFees(
                 txInfo, selectedNetwork, networkSpotPrice, solFeeEstimatesFee);
         TxDataUnion txDataUnion = txInfo.txDataUnion;
-        TxData1559 txData = txInfo.txDataUnion.which() == TxDataUnion.Tag.EthTxData1559
-                ? txInfo.txDataUnion.getEthTxData1559()
+        TxData1559 txData = txDataUnion.which() == TxDataUnion.Tag.EthTxData1559
+                ? txDataUnion.getEthTxData1559()
                 : null;
-        SolanaTxData solTxData = txInfo.txDataUnion.which() == TxDataUnion.Tag.SolanaTxData
-                ? txInfo.txDataUnion.getSolanaTxData()
+        SolanaTxData solTxData = txDataUnion.which() == TxDataUnion.Tag.SolanaTxData
+                ? txDataUnion.getSolanaTxData()
                 : null;
         ;
-        FilTxData filTxData = txInfo.txDataUnion.which() == TxDataUnion.Tag.FilTxData
-                ? txInfo.txDataUnion.getFilTxData()
+        FilTxData filTxData = txDataUnion.which() == TxDataUnion.Tag.FilTxData
+                ? txDataUnion.getFilTxData()
                 : null;
-        ;
```

Comment:
:) thanks for cleaning this but the previous one on line 149 slipped through

Reply:
That is intentional, it is just an extra unwanted semi-colon, so removed it :)

Reply:
will remove it :) thanks